exporter/loadbalancingexporter/README.md

```diff
@@ -162,6 +166,48 @@ service:
         - loadbalancing
 ```
 
+The DNSSRVNOA Resolver is useful in situations when you want to return hostnames instead of IPs for endpoints. An example would be a `StatefulSet`-backed headless kubernetes `Service` with istio.
+
+The format for the name of an SRV record is `_service._proto.name` such as `_ldap._tcp.example.com`. The full record contains:
```

Comment:
I'll def take a look at making this clearer but wanted to answer your questions so you can properly gauge if you think my edits work.

The first paragraph is more or less describing the format of SRV records. Most of it is taken from the rfc itself. I personally feel like using SRV records is a less basic approach and a person likely knows about them already. Maybe instead of explaining it the doc could benefit from a general description of scenarios when you might choose one of the resolvers over another?

The "NOA" means "No A record" as in do not have the app perform 2 levels of DNS resolutions. Instead of IP addresses, A records are returned so it's the difference between `curl www.google.com` vs `curl 142.250.176.4`.

Though I've heard of some gRPC client load balancing advantages, I'm mostly implementing the NOA for istio. You need something in the ".svc.cluster.local" (or equivalent `Service` domain,) for istio to work properly and resolving just the SRV record and allowing the kernel to resolve the A records makes it work. If all it has is an IP it will not use mTLS and the endpoints you're trying to talk to will reject the traffic.